ecole-doctorale-de-sciences-po-author-date-french.csl

```diff
@@ -0,0 +1,440 @@
+    </choose>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <name form="long" and="text" delimiter-precedes-last="never" sort-separator=", " font-style="normal">
+        <name-part name="family" font-variant="small-caps"/>
+      </name>
+      <label form="short" prefix="&#160;(" suffix=".)"/>
+    </names>
+  </macro>
+<macro name="translator">
+        <names variable="translator">
+            <name form="long" and="text" delimiter-precedes-last="never" sort-separator=" " font-style="normal" prefix="traduit par ">
+				<name-part name="family" font-variant="small-caps" />
+            </name>
+        </names>
+    </macro>
+<macro name="archive">
+    <choose>
+      <if variable="archive archive_location" match="any">
+        <group delimiter=". ">
+          <text variable="archive_location" text-case="capitalize-first"/>
+          <text variable="archive"/>
+          <text variable="archive-place"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+<macro name="scale">
+<choose>
+<if type="map" match="any">
+       <text variable="scale" prefix="Echelle "/>
+</if>
+</choose>
+    </macro>
+<macro name="medium">
+<choose>
+<if type="graphic motion_picture" match="any">
+       <text variable="medium"/>
+</if>
+</choose>
+    </macro>
+<macro name="number">
+<choose>
+<if type="patent" match="any">
+       <text variable="number" prefix="Brevet "/>
+   </if>
+<else-if type="report" match="any">
+<text variable="number" prefix="Rapport "/>
+      </else-if>
+   </choose>
+</macro>
+<macro name="dimensions">
+<choose>
+<if type="motion_picture graphic broadcast" match="any">
+       <text variable="dimensions"/>
+   </if>
+   </choose>
+</macro>
+<macro name="originally-published">
+      <date variable="original-date" form="text" date-parts="year" prefix="[" suffix="] "/>
+  </macro>
+<macro name="version">
+<choose>
+<if type="book" match="any">
+       <text variable="version" prefix="Version "/>
+   </if>
+   </choose>
+</macro>
+<macro name="access">
+    <choose>
+      <if type="article article-journal article-magazine article-newspaper book broadcast chapter entry-dictionary entry-encyclopedia thesis graphic legal_case legislation manuscript map motion_picture paper-conference patent post post-weblog personal_communication report song speech webpage" match="any">
+      <text variable="URL" prefix="En ligne : " suffix=" "/>          
+    <text value="consulté le" prefix="[" suffix=" "/>
+       <date variable="accessed" form="text" suffix="">
+        <date-part name="day"/>
+            <date-part name="month"/>
+            <date-part name="year"/>
+          </date>
+      </if>
+    </choose>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="bill book broadcast graphic legal_case legislation map manuscript motion_picture patent personal_communication report song" match="any">
+        <text variable="title" text-case="capitalize-first" font-style="italic"/>
```

Comment:
Any reason for all the `capitalize-first` on titles? It would normally do very little and if people have a title starting with a lowercase letter, that's presumably on purpose...